src/gameplay/maps/systems.rs

```diff
@@ -16,6 +16,49 @@ pub fn spawn_oob_colliders(
     if let Some(layer_data) = layer_query.iter().next() {
         println!("LAYER METADATA");
         println!("{:?}", layer_data);
+        let grid_size = layer_data.grid_size as f32;
+        let height = layer_data.c_hei as f32;
+        let width = layer_data.c_wid as f32;
+
+        // left wall
+        commands.spawn((
```

Comment:
Would you want me to refactor this spawn into a helper function?

Reply:
nope! this is perfect